src/ir_Corona.cpp

```diff
@@ -36,15 +38,15 @@ const uint8_t kCoronaTolerance = 5;  // +5%
 #if SEND_CORONA_AC
 /// Send a CoronaAc formatted message.
 /// Status: STABLE / Working on real device.
-/// Where data is:
-///   uint8_t data[kCoronaAcStateLength] = {
-///   0x28, 0x61, 0x3D, 0x19, 0xE6, 0x37, 0xC8,
-///   0x28, 0x61, 0x6D, 0xFF, 0x00, 0xFF, 0x00,
-///   0x28, 0x61, 0xCD, 0xFF, 0x00, 0xFF, 0x00};
-///
 /// @param[in] data An array of bytes containing the IR command.
 /// @param[in] nbytes Nr. of bytes of data in the array.
-///                   (>=kCoronaAcStateLength)
+/// e.g.
+/// @code
+///   uint8_t data[kCoronaAcStateLength] = {
+///       0x28, 0x61, 0x3D, 0x19, 0xE6, 0x37, 0xC8,
+///       0x28, 0x61, 0x6D, 0xFF, 0x00, 0xFF, 0x00,
+///       0x28, 0x61, 0xCD, 0xFF, 0x00, 0xFF, 0x00};
+/// @endcode
 /// @param[in] repeat Nr. of times the message is to be repeated.
 void IRsend::sendCoronaAc(const uint8_t data[],
```

Comment:
Maybe a revisit on the code that auto_analyse generates to be sure it requires as little maintenance as possible

Reply:
I never really meant those/that code comment to actually make it into the `src/` tree (from `auto_analyse`). It was more a "FYI" for folks new to the library.

Reply:
I.e. I'd rather leave it in the python prog to aid users so they can help themselves, but remove it if/when they submit a PR etc. Thoughts?

Reply:
I find it useful from a consumer perspective, when reading the code and trying to understand what functions do, and examples of how to call them is super useful for newcomers.

At the same time I understand that it can bee seen as unnecessary for those that are used to the code. but fact is that many that look at this library, does so for the first time, (and many so without knowledge about .h files, or an editor without any hints at all.)

And even if it is only one sample of data, it does aid in understanding the parts of it.

Reply:
Okay, you've convinced me. Feel free to pretty up the code output of `auto_analyse` and we will include it from now on. More documentation == better.